### ISSUE TITLE
Sticky columns are broken by incell editing with column editor templates

### ISSUE THREAD

**Issue by al-beqaa** [telerik/kendo-ui-core#7861]
### Reproduction of the problem
Create a scrollable grid with incell editing, a save changes toolbar button, a 
sticky column, and a column with any editor template. Edit the column with the template, then save changes to the grid. Scroll horizontally. You should see that the columns for the edited row are no longer sticky.

On inspection it looks like the left and top styles calculated for the sticky items are saved as data-style-left... etc and not actually applied to the style attribute. Saving changes again fixes it.

Example: https://dojo.telerik.com/avEteNUc

### Expected/desired behavior
Sticky columns should stay sticky.

### Environment

* **Kendo UI version:** 2024.2.514, 2024.1.319
* **jQuery version:** 3.6
* **Browser:** Tested on both Chrome and Firefox

**Comment by al-beqaa** [comment]
I have found this works as a temporary solution for anyone else having this problem:

https://dojo.telerik.com/uhoSajav

It feels a bit clumsy, forgive me as I'm not well-versed in the relationships between Kendo events.

**Comment by gdenchevprog** [comment]
> I have found this works as a temporary solution for anyone else having this problem:
> 
> https://dojo.telerik.com/uhoSajav
> 
> It feels a bit clumsy, forgive me as I'm not well-versed in the relationships between Kendo events.

Thank you for sharing a workaround @al-beqaa ! The suggested approach is completely valid.

TL;DR; of the bug - a couple of the kendo styles are not re-evaluated upon refresh and this causes the cell to not stick properly. Calling the refresh() method will resolve the problem.

In case you do not wish to perform a full grid refresh(due to performance problems or any other reason), you can re-evaluate the styles of only the updated row.

```javascript
      function onDataBound(e) {
        let grid = e.sender,
            tbody = e.sender.tbody,
            stylesToParse = tbody.find("[data-style-left]");
        
        if (stylesToParse.length) {
          kendo.applyStylesFromKendoAttributes(stylesToParse, ["left", "right"]);
        }
      }
```

https://dojo.telerik.com/@gdenchev/ELicuHOB
